README.md

```diff
@@ -70,9 +70,45 @@ should look something like this:
 -DVCPKG_TARGET_TRIPLET=x64-osx-static
 ```
 
+See the following images:
+
+### CLion ToolChain Settings
+
+![toolchain](images/toolchains.png?raw=true "Toolchains Settings")
```

Comment:
Annoyance but can you put these images on the "images issue" https://github.com/mongodb/genny/issues/296? Just add them as a comment and grab the URL. This saves us from having binary assets that get pulled on every clone. 

These images are small, but it'll add up (and it makes it easier to add more images if you don't have to think much about image sizes)

Reply:
Will do.